eng/pipelines/runtime.yml

```diff
@@ -75,31 +76,81 @@ jobs:
     - OSX_x64
     - Linux_x64
     - Linux_arm
+    - Linux_arm64
+    - Linux_musl_x64
     - Windows_NT_x86
     - Windows_NT_x64
     - Windows_NT_arm
     - Windows_NT_arm64
     jobParameters:
       testGroup: innerloop
+      condition: eq(dependencies.checkout.outputs['SetPathVars_coreclr.containsChange'], true)
 
 #
-# Build libraries using live CoreLib from CoreCLR
+# Build CoreCLR debug
+# Only when it contains coreclr changes
```

Comment:
May want to expand this comment to mention the debug CoreCLR is used for testing only. Hence it only makes sense to build when we're going to be running the tests that depend on it.

Reply:
Good idea.

Reply:
I do not think it makes sense to run to run tests on debug CoreCLR in the CI. Testing on checked build provides 99.9% same coverage and it is much cheaper.

Reply:
Ok, makes sense. Thanks for the input, I will remove it.